website/markdown/docs/links.mdx

```diff
@@ -33,7 +33,8 @@ import DocTitle from './components/doc-title'
 - [Sign your apps](/docs/commands/signing/)
 - [Build schemes](/docs/commands/build/)
 - [Scaffold files](/docs/commands/scaffold/)
-- [Lint projects](/docs/commands/lint/)
+- [Lint projects](/docs/commands/lint-project/)
```

Comment:
What about keeping everything in the same document, `lint.mdx` and then creating subsections?

Reply:
Doc have been merged. Please check `excerpt` value in `lint.mdx` I am not sure if I defined it correctly.